srk-nixpkgs.nix

```diff
@@ -1,189 +1,54 @@
 { pkgs ? (import <nixpkgs> {})
 , compiler ? pkgs.haskell.packages.ghc802
-, genesisN ? 3 
-, slotDuration ? 20 
-, networkDiameter ? 6
-, mpcRelayInterval ? 16 } :
```

Comment:
I guess it's not urgent, just asked because thought you had been working with making our configs more extensible. Let's close this discussion for now. It's pretty easy to solve the problem if it arises in hour or so.